templates/web/northamptonshire/report/new/roads_message.html

```diff
@@ -0,0 +1,15 @@
+<div id="js-roads-responsibility" class="box-warning hidden">
```

Comment:
Equivalent messages on .com need to be made presumably so that they display there, currently just getting "not maintained". Check doesn't interfere with Bucks similar.